packages/console/src/pages/ApplicationDetails/components/Settings.tsx

```diff
@@ -105,7 +105,12 @@ const Settings = ({ oidcConfig }: Props) => {
           control={control}
           defaultValue={[]}
           rules={{
-            validate: createValidatorForRhf(uriPatternRules),
+            validate: createValidatorForRhf({
+              pattern: {
+                verify: uriValidator(false, true),
```

Comment:
what does the `false, true` mean? Would be better to give it a readable property name. E.g.
```
uriValidator({ allowBlank: true, validateAsUrlOrigin: true })
```